README.md

```diff
@@ -1,7 +1,42 @@
 # pxe-server-dev
 
-Start server:
+## Requirements
+
+```
+sudo modprobe nfs nfsd
+```
+
+## Start
 
 ```
 ./up.sh
 ```
+
+## Possible issues
+
+* `rpc_pipefs`
+
+```
+nfs-server    | ==================================================================
+nfs-server    |       STARTING SERVICES ...
+nfs-server    | ==================================================================
+nfs-server    | ----> mounting rpc_pipefs filesystem onto /var/lib/nfs/rpc_pipefs
+nfs-server    | mount: mounting rpc_pipefs on /var/lib/nfs/rpc_pipefs failed: Permission denied
+nfs-server    | ---->
+nfs-server    | ----> ERROR: unable to mount rpc_pipefs filesystem onto /var/lib/nfs/rpc_pipefs
+nfs-server    | ---->
+```
+
+See [AppArmor configuration](https://github.com/ehough/docker-nfs-server/blob/develop/doc/feature/apparmor.md#apparmor)
```

Comment:
Sometimes abstractions/lxc/container-base may be missing when adding apparmor profile. @macpijan could You add information to install lxc package in such case?

Reply:
https://github.com/ehough/docker-nfs-server/pull/19/files